src/js/directives/image-list-directive.test.js

```diff
@@ -89,6 +115,18 @@ describe('Image List Directive', function () {
 
     });
 
+      it('should set a class of img-responsive for each IMG that matches the path in the array', function () {
+        var elm = compileDirective(defaultTemplate, scope);
+        var imgClass;
+
+        angular.forEach(scope.mockImages, function(url, index){
+            imgClass = elm.find('ul li img').eq(index);
+            expect(imgClass.attr('class')).to.equal('img-responsive');
+        });
+
+    });
+
   });
 
-});
\ No newline at end of file
+});
+
```

Comment:
Can you check the indentation on this file? Looks like a few blocks are off a bit.